Auto update failing for ipset

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [ipset](https://github.com/termux/termux-packages/blob/b6238f1f8cf7bf191e27a9233407292aa85fc445/root-packages/ipset) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating ipset [Current version: 7.19]
INFO: package being updated to 7.20.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  671k  100  671k    0     0   500k      0  0:00:01  0:00:01 --:--:--  501k

You are about to commit these changes:

--------------------
diff --git a/root-packages/ipset/build.sh b/root-packages/ipset/build.sh
index 4404f3d89..f4878f2b8 100644
--- a/root-packages/ipset/build.sh
+++ b/root-packages/ipset/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://ipset.netfilter.org
 TERMUX_PKG_DESCRIPTION="Administration tool for kernel IP sets"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="Vishal Biswas @vishalbiswas"
-TERMUX_PKG_VERSION="7.19"
+TERMUX_PKG_VERSION="7.20"
 TERMUX_PKG_SRCURL=https://ipset.netfilter.org/ipset-${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=9bc1fba48d65786e3e0b63dc6b669a866823d77840c6990c0c6b23078ec2c4d6
+TERMUX_PKG_SHA256=6ae85b1d81832c2b67df7330d3671a3462e39ec98328056a18a29d44c27709e5
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libmnl"
 TERMUX_PKG_BREAKS="ipset-dev"
--------------------

bump(root/ipset): 7.20


Not committing to Git!

INFO: Trying to build package.
INFO: Package ipset exists in root repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building ipset for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  34368      0 --:--:-- --:--:-- --:--:-- 34379
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2031      0 --:--:-- --:--:-- --:--:--  2031
100   833  100   833    0     0   2030      0 --:--:-- --:--:-- --:--:--  2026
gpg: Signature made Fri 02 Feb 2024 12:16:15 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  35567      0 --:--:-- --:--:-- --:--:-- 35612
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2263      0 --:--:-- --:--:-- --:--:--  2269
gpg: Signature made Thu 01 Feb 2024 07:15:04 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  35644      0 --:--:-- --:--:-- --:--:-- 35649
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2215      0 --:--:-- --:--:-- --:--:--  2221
gpg: Signature made Thu 01 Feb 2024 06:24:03 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libmnl@1.0.5 if necessary...
Found libmnl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libm/libmnl/libmnl_1.0.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8716  100  8716    0     0  22498      0 --:--:-- --:--:-- --:--:-- 22521
extracting libmnl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@28-3 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20231001-3 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20231001-3 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20231001-3 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20231001-3 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgmp@6.3.0 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency coreutils@9.4 if necessary...
Found coreutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting coreutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.59.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2023.12.12 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2023.12.12 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.2.0 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.2.0 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.0 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.6.0 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.6.0 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dash@0.5.12 if necessary...
Found dash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dialog@1.3-20230209-0 if necessary...
Found dialog in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dialog to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency diffutils@3.10 if necessary...
Found diffutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting diffutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency findutils@4.9.0-2 if necessary...
Found findutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting findutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmpfr@4.2.1 if necessary...
Found libmpfr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmpfr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gawk@5.3.0 if necessary...
Found gawk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gawk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.42 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.42 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency grep@3.11 if necessary...
Found grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gzip@1.13 if necessary...
Found gzip in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gzip to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency less@633-1 if necessary...
Found less in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting less to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency procps@3.3.17-5 if necessary...
Found procps in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting procps to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency psmisc@23.6-2 if necessary...
Found psmisc in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting psmisc to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sed@4.9-1 if necessary...
Found sed in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sed to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-glob@0.6-2 if necessary...
Found libandroid-glob in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-glob to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tar@1.35 if necessary...
Found tar in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tar to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-am@0.8.0 if necessary...
Found termux-am in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-am to /home/builder/.termux-build/_cache-all...
Downloading dependency termux-am-socket@1.5.0 if necessary...
Found termux-am-socket in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-am-socket to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-exec@1:1.0 if necessary...
Found termux-exec in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-exec to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libblkid@2.39.3-3 if necessary...
Found libblkid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libblkid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmount@2.39.3-3 if necessary...
Found libmount in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmount to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsmartcols@2.39.3-3 if necessary...
Found libsmartcols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsmartcols to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libuuid@2.39.3-3 if necessary...
Found libuuid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libuuid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libfdisk@2.39.3-3 if necessary...
Found libfdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libfdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcap-ng@2:0.8.4 if necessary...
Found libcap-ng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcap-ng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency util-linux@2.39.3-3 if necessary...
Found util-linux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting util-linux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency uuid-utils@2.39.3-3 if necessary...
Found uuid-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting uuid-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency mount-utils@2.39.3-3 if necessary...
Found mount-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting mount-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fdisk@2.39.3-3 if necessary...
Found fdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency blk-utils@2.39.3-3 if necessary...
Found blk-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting blk-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.4.6 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.4.6 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-tools@1.40.5-1 if necessary...
Found termux-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-tools to /home/builder/.termux-build/_cache-all...
Downloading dependency bash@5.2.26 if necessary...
Found bash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtool@2.4.7-1 if necessary...
Found libtool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libtool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libltdl@2.4.7-1 if necessary...
Found libltdl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libltdl to /home/builder/.termux-build/_cache-aarch64...
Downloading https://ipset.netfilter.org/ipset-7.20.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  2  671k    2 16384    0     0  24656      0  0:00:27 --:--:--  0:00:27 24637
100  671k  100  671k    0     0   626k      0  0:00:01  0:00:01 --:--:--  626k
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls
checking build system type... x86_64-pc-linux-gnu
checking host system type... aarch64-unknown-linux-android
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-linux-android-strip... llvm-strip
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking how to create a pax tar archive... gnutar
checking whether make supports nested variables... (cached) yes
checking whether ln -s works... yes
checking how to print strings... printf
checking whether make supports the include directive... yes (GNU style)
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to accept ISO C89... none needed
checking whether aarch64-linux-android-clang understands -c and -o together... yes
checking dependency style of aarch64-linux-android-clang... none
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by aarch64-linux-android-clang... ld.lld
checking if the linker (ld.lld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... llvm-nm
checking the name lister (llvm-nm) interface... BSD nm
checking the maximum length of command line arguments... 3145728
checking how to convert x86_64-pc-linux-gnu file names to aarch64-unknown-linux-android format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for ld.lld option to reload object files... -r
checking for aarch64-linux-android-objdump... llvm-objdump
checking how to recognize dependent libraries... pass_all
checking for aarch64-linux-android-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for aarch64-linux-android-ar... llvm-ar
checking for archiver @FILE support... @
checking for aarch64-linux-android-strip... (cached) llvm-strip
checking for aarch64-linux-android-ranlib... llvm-ranlib
checking command to parse llvm-nm output from aarch64-linux-android-clang object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for aarch64-linux-android-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking how to run the C preprocessor... aarch64-linux-android-cpp
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if aarch64-linux-android-clang supports -fno-rtti -fno-exceptions... yes
checking for aarch64-linux-android-clang option to produce PIC... -fPIC -DPIC
checking if aarch64-linux-android-clang PIC flag -fPIC -DPIC works... yes
checking if aarch64-linux-android-clang static flag -static works... yes
checking if aarch64-linux-android-clang supports -c -o file.o... yes
checking if aarch64-linux-android-clang supports -c -o file.o... (cached) yes
checking whether the aarch64-linux-android-clang linker (ld.lld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... Android linker
checking how to hardcode library paths into programs... immediate
checking for shl_load... no
checking for shl_load in -ldld... no
checking for dlopen... yes
checking whether a program can dlopen itself... cross
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking what extension is used for runtime loadable modules... .so
checking what variable specifies run-time module search path... LD_LIBRARY_PATH
checking for the default library search path... /lib /usr/lib
checking for library containing dlopen... none required
checking for dlerror... yes
checking for shl_load... (cached) no
checking for shl_load in -ldld... (cached) no
checking for dld_link in -ldld... no
checking for _ prefix in compiled symbols... no
checking whether deplibs are loaded by dlopen... yes
checking for argz.h... no
checking for error_t... no
checking for argz_add... no
checking for argz_append... no
checking for argz_count... no
checking for argz_create_sep... no
checking for argz_insert... no
checking for argz_next... no
checking for argz_stringify... no
checking whether libtool supports -dlopen/-dlpreopen... yes
checking for ltdl.h... yes
checking whether lt_dlinterface_register is declared... yes
checking for lt_dladvise_preload in -lltdl... yes
checking where to find libltdl headers... 
checking where to find libltdl library... -lltdl
checking for unistd.h... (cached) yes
checking for dl.h... no
checking for sys/dl.h... no
checking for dld.h... no
checking for mach-o/dyld.h... no
checking for dirent.h... yes
checking for closedir... yes
checking for opendir... yes
checking for readdir... yes
checking for strlcat... yes
checking for strlcpy... yes
checking pkg-config is at least version 0.9.0... yes
checking for aarch64-linux-android-gcc... (cached) aarch64-linux-android-clang
checking whether we are using the GNU C compiler... (cached) yes
checking whether aarch64-linux-android-clang accepts -g... (cached) yes
checking for aarch64-linux-android-clang option to accept ISO C89... (cached) none needed
checking whether aarch64-linux-android-clang understands -c and -o together... (cached) yes
checking dependency style of aarch64-linux-android-clang... (cached) none
checking whether ln -s works... yes
checking for libmnl >= 1... yes
checking whether NLA_F_NESTED is declared... yes
checking whether NLA_F_NET_BYTEORDER is declared... yes
checking whether NLA_TYPE_MASK is declared... yes
checking for union nf_inet_addr... yes
checking for gethostbyname2... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating include/libipset/Makefile
config.status: creating lib/Makefile
config.status: creating lib/libipset.pc
config.status: creating src/Makefile
config.status: creating utils/Makefile
config.status: creating kernel/include/linux/netfilter/ipset/ip_set_compat.h
config.status: creating kernel/include/linux/netfilter/ipset/ip_set_compiler.h
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls

ipset userspace tool configuration:
    Dynamic module loading: disabled
    Static modules:
        ipset_bitmap_ip
        ipset_bitmap_ipmac
        ipset_bitmap_port
        ipset_hash_ip
        ipset_hash_ipmac
        ipset_hash_ipmark
        ipset_hash_ipport
        ipset_hash_ipportip
        ipset_hash_ipportnet
        ipset_hash_mac
        ipset_hash_net
        ipset_hash_netiface
        ipset_hash_netnet
        ipset_hash_netport
        ipset_hash_netportnet
        ipset_list_set
    Dynamic modules:
make  all-recursive
make[1]: Entering directory '/home/builder/.termux-build/ipset/build'
Making all in include/libipset
make[2]: Entering directory '/home/builder/.termux-build/ipset/build/include/libipset'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/builder/.termux-build/ipset/build/include/libipset'
Making all in lib
make[2]: Entering directory '/home/builder/.termux-build/ipset/build/lib'
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_bitmap_ip_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_bitmap_ip.lo /home/builder/.termux-build/ipset/src/lib/ipset_bitmap_ip.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_bitmap_ipmac_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_bitmap_ipmac.lo /home/builder/.termux-build/ipset/src/lib/ipset_bitmap_ipmac.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_bitmap_port_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_bitmap_port.lo /home/builder/.termux-build/ipset/src/lib/ipset_bitmap_port.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_ip_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_ip.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_ip.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_ipport_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_ipport.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_ipport.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_ipmark_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_ipmark.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_ipmark.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_ipportip_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_ipportip.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_ipportip.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_ipportnet_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_ipportnet.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_ipportnet.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_net_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_net.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_net.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_netnet_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_netnet.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_netnet.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_netportnet_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_netportnet.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_netportnet.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_netport_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_netport.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_netport.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_netiface_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_netiface.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_netiface.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_ipmac_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_ipmac.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_ipmac.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_hash_mac_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_hash_mac.lo /home/builder/.termux-build/ipset/src/lib/ipset_hash_mac.c
/bin/bash ../libtool --silent --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ipset/src/lib -I..    -I/home/builder/.termux-build/ipset/src/include -I/data/data/com.termux/files/usr/include -std=gnu99  -O2 -DNDEBUG -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -D_INIT=ipset_list_set_init \
	 -DTYPE_INCLUSIVE\
	-c -o ipset_list_set.lo /home/builder/.termux-build/ipset/src/lib/ipset_list_set.c
  CC       args.lo
  CC       data.lo
  CC       errcode.lo
  CC       icmp.lo
  CC       icmpv6.lo
  CC       list_sort.lo
  CC       mnl.lo
  CC       parse.lo
  CC       print.lo
  CC       session.lo
  CC       types.lo
/home/builder/.termux-build/ipset/src/lib/session.c:1089:37: warning: data argument not used by format string [-Wformat-extra-args]
                                safe_snprintf(session, "\",\n", arg->name[0]);
                                                       ~~~~~~~  ^
/home/builder/.termux-build/ipset/src/lib/session.c:1094:34: warning: data argument not used by format string [-Wformat-extra-args]
                        safe_snprintf(session, ",\n", arg->name[0]);
                                               ~~~~~  ^
  CC       ipset.lo
  GEN      types_init.c
  CC       types_init.lo
2 warnings generated.
  CCLD     libipset.la
ld.lld: error: version script assignment of 'LIBIPSET_1.0' to symbol 'ipset_parse_name' failed: symbol not defined
ld.lld: error: version script assignment of 'LIBIPSET_1.0' to symbol 'ipset_session_error' failed: symbol not defined
ld.lld: error: version script assignment of 'LIBIPSET_1.0' to symbol 'ipset_session_warning' failed: symbol not defined
ld.lld: error: version script assignment of 'LIBIPSET_1.0' to symbol 'ipset_debug_msg' failed: symbol not defined
ld.lld: error: version script assignment of 'LIBIPSET_1.0' to symbol 'ipset_shift_argv' failed: symbol not defined
ld.lld: error: version script assignment of 'LIBIPSET_3.0' to symbol 'ipset_session_outfn' failed: symbol not defined
clang-17: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [Makefile:656: libipset.la] Error 1
make[2]: Leaving directory '/home/builder/.termux-build/ipset/build/lib'
make[1]: *** [Makefile:594: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/ipset/build'
make: *** [Makefile:521: all] Error 2
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-02-02 18:13:05 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/7759779792">7759779792</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/root-packages/ipset/build.sh
+++ b/root-packages/ipset/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://ipset.netfilter.org
 TERMUX_PKG_DESCRIPTION="Administration tool for kernel IP sets"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="Vishal Biswas @vishalbiswas"
-TERMUX_PKG_VERSION="7.19"
+TERMUX_PKG_VERSION="7.20"
 TERMUX_PKG_SRCURL=https://ipset.netfilter.org/ipset-${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=9bc1fba48d65786e3e0b63dc6b669a866823d77840c6990c0c6b23078ec2c4d6
+TERMUX_PKG_SHA256=6ae85b1d81832c2b67df7330d3671a3462e39ec98328056a18a29d44c27709e5
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libmnl"
 TERMUX_PKG_BREAKS="ipset-dev"
@@ -13,3 +13,8 @@ TERMUX_PKG_BUILD_DEPENDS="libtool"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --with-kmod=no
 "
+
+termux_step_pre_configure() {
+	# Workaround for version script error
+	LDFLAGS+=" -Wl,--undefined-version"
+}

